js/shop.js

```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
```

Comment:
Yup, verified the term id with Jill, for both HDS and COLGSAS.  From talking to Colin in devops channel, it's my understanding that we should check in the final product to the school-specific repos they belong in, and the only processing devops is doing on deploy is concatenation/minification.